modules/installation-upi-machines-iso.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-upi-machines-iso-{context}"]
+= Creating {op-system-first} machines using an ISO image
+
+Before you install a user provisioned infrastructure (UPI) cluster on bare
+metal, you need to create {op-system} machines for it to use. You can use an
+ISO image to create the machines.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Upload the control plane, compute, and bootstrap Ignition config files that the
+installation program created to your HTTP server. Note the URLs of these files.
+
+. Obtain the {op-system} ISO image:
```

Comment:
@imcleod, will you PTAL? Does this section contain all of the parameters that you need to set to install RHCOS?

Reply:
This provisionally looks good to me for ISO.

Be aware that we also offer PXE/network installation.  Unclear to me if that is meant to be covered elsewhere or should be added here.

Am going to ask @arithx to chime in as well.

Reply:
I'm making a follow-up JIRA to add the PXE booting instructions as an improvement to address after beta4 goes out.